src/core/bidi.js

```diff
@@ -17,17 +17,14 @@
 
 (function (root, factory) {
   if (typeof define === 'function' && define.amd) {
-    define('pdfjs/core/bidi', ['exports', 'pdfjs/shared/global'], factory);
+    define('pdfjs/core/bidi', ['exports'], factory);
   } else if (typeof exports !== 'undefined') {
-    factory(exports, require('../shared/global.js'));
+    factory(exports);
   } else {
-    factory((root.pdfjsCoreBidi = {}), root.pdfjsSharedGlobal);
+    factory((root.pdfjsCoreBidi = {}));
   }
-}(this, function (exports, sharedGlobal) {
+}(this, function (exports) {
 
-var PDFJS = sharedGlobal.PDFJS;
-
-var bidi = PDFJS.bidi = (function bidiClosure() {
```

Comment:
Now that this is gone, will the `export` line at the end of the file still work?